app.py

```diff
@@ -360,6 +372,11 @@ async def hasRole(member: discord.Member, role_name):
             return True
     return False
 
+async def getRoleId(role_name: str):
+    guild = bot.get_guild(GUILD_ID)
+    role = discord.utils.get(guild.roles, name = role_name)
```

Comment:
```suggestion
    role = discord.utils.get(guild.roles, name=role_name)
```

Reply:
bro who cares abt that it doesn't matter lol

Reply:
It is just for consistency.